src/nvim/ex_cmds2.c

```diff
@@ -116,6 +116,11 @@ struct source_cookie {
 static int debug_greedy = FALSE;        /* batch mode debugging: don't save
                                            and restore typeahead. */
 
+static int get_maxbacktrace_level(void);
+static void do_setdebugtracelevel(char_u *arg);
+static void do_checkbacktracelevel(void);
+static void do_showbacktrace(char_u *cmd);
```

Comment:
These forward declarations aren't needed because of this line:

```
#ifdef INCLUDE_GENERATED_DECLARATIONS
# include "ex_cmds2.c.generated.h"
#endif
```